src/components/BugReportDialog.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <Dialog :visible="visible" @update:visible="$emit('hide')">
+        <template #header>
+            <h3>Report a bug</h3>
+        </template>
+        <div class="flex flex-column pt-4">
+            <span class="p-float-label mb-5">
+            <InputText
+                class="w-full"
+                id="name-input"
+                type="text"
+                v-model="name"
+            />
+            <label for="name-input">Bug name</label>
+            </span>
+
+            <span class="p-float-label mb-2">
+            <Textarea
+                class="w-full"
+                id="desc-input"
+                v-model="description"
+                rows="8"
+                cols="50"
+            />
+            <label for="desc-input">Bug description</label>
+            </span>
+        </div>
+        <template #footer>
+            <Button label="Send" icon="pi pi-check" @click="report" :loading="loading"/>
+        </template>
+    </Dialog>
+</template>
+
+<script>
+
+import Dialog from "primevue/dialog";
+import Button from "primevue/button";
+import InputText from "primevue/inputtext";
+import Textarea from "primevue/textarea";
+
+import { ref } from "vue";
+import { reportBug } from "../services/api";
+import { useToast } from "primevue/usetoast";
+
+export default {
```

Comment:
In other components are short-syntax setup scripts and here u are using this long-syntax setup. Use everywhere only one syntax (imo this shorter is beter)